core/src/main/java/io/aiven/klaw/repository/AclRequestsRepo.java

```diff
@@ -29,7 +29,7 @@ List<Object[]> findAllRecordsCountForTeamId(
 
   @Query(
       value =
-          "select count(*) from kwaclrequests where (requestor = :userId or approver = :userId) and tenantid = :tenantId",
+          "select count(*) from kwaclrequests where (requestor = :userId or approver = :userId) and tenantid = :tenantId and topicstatus='created'",
```

Comment:
Do we need to worry about them being an approver? Maybe if they were the only approver but Im not sure we do need to worry about that?

Reply:
i think we should ignore them too, so its their responsibility to setup approvers